.github/workflows/test.yml

```diff
@@ -0,0 +1,173 @@
+name: ovn-ci
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
@dave-tucker just a note I've been playing around with adding this:
name: Go
on:
 push:
 pull_request:
 pull_request_review_comment:
   types:
    - created

And then in under the jobs doing:
verify:
    name: Verify
    runs-on: ubuntu-latest
    if: github.event_name == 'pull_request' || github.event_name == 'push' || github.event.pull_request_review_comment.comment.body == '/retest'

It doesn't seem to work though from me trying it out. Maybe you know how to fix it? I've also tried removing the if statement and just triggering on a PR comment created, but that didn't trigger anything either

Reply:
If you want to implement slash commands I'd recommend that we do that in a new workflow as the logic can get quite complicated. To do this (sudo code):

```
run the workflow on issue_comment event
if the body of the comment is exactly '/retest' and the issue is a pull request then
  trigger the ci workflow to for pr $number using the github api
```

Reply:
it doesnt have to be a slash command, that's just the trigger we use in openshift. It's not issue_comment event, that is for github issue I think we want pull_request_review_comment. Yeah we can make a new issue for this and track it:
https://github.com/ovn-org/ovn-kubernetes/issues/1216

I've tried just triggering on delete or create (ignoring body, but it doesnt seem to work)

Reply:
ack'd. continuing discussion in that issue